extensions/wikia/AdEngine/js/SevenOneMediaHelper.js

```diff
@@ -39,7 +39,7 @@ define('ext.wikia.adEngine.sevenOneMediaHelper', [
 			},
 			'fullbanner2': {
 				SOI_FB2: true,
-				SOI_PB: true,    // powerbanner (728x180)
+				SOI_PB: false,   // powerbanner (728x180)
```

Comment:
`SOI_PB`, are you sure? It looks like `SOI_PD` should've been changed to `false`.